src/share/classes/com/sun/javatest/regtest/tool/Tool.java

```diff
@@ -2336,12 +2339,20 @@ public enum ReportMode { NONE, EXECUTED, FILES, ALL_EXECUTED, ALL }
     private static final String MANUAL    = "manual";
 
     private static final String[] DEFAULT_UNIX_ENV_VARS = {
-        "DISPLAY", "GNOME_DESKTOP_SESSION_ID", "HOME", "LANG",
-        "LC_ALL", "LC_CTYPE", "LPDEST", "PRINTER", "TZ", "XMODIFIERS"
+            "DBUS_SESSION_BUS_ADDRESS", "DISPLAY",
```

Comment:
Any reason you did not include these below from the bug report ?
They aren't known to be critical but ..
DESKTOP_SESSION=ubuntu-wayland 
GDMSESSION=ubuntu-wayland 
GNOME_SHELL_SESSION_MODE=ubuntu

Reply:
> Any reason you did not include these below from the bug report ?
> They aren't known to be critical but ..
> DESKTOP_SESSION=ubuntu-wayland
> GDMSESSION=ubuntu-wayland
> GNOME_SHELL_SESSION_MODE=ubuntu

Sorry, I was going according to our private side-conversation.